### ISSUE TITLE
Support for requesting permission

### ISSUE THREAD

**Issue by diegotrp** [sagiegurari/simple-web-notification#5]
#### Problem Description
Hi, very nice library! I understand that currently the library doesn't support requesting for permission without the need of trying to show a notification (i am trying to ask for permission when the page loads).  I thought about showing a notification for a short time but id didn't work, so i ended up using the native way of doing it.
````js
Notification.requestPermission();
````
Is there going to be support for it in the future? Thanks in advance.

**Comment by sagiegurari** [comment]
I'm not 100% sure its great UX to annoy the user about permissions if you don't need it, but I do see many sites doing it.
I can modify the code to expose that api, however it would basically be the same as what you did, so i'm not sure if that would help much.

**Comment by diegotrp** [comment]
Yes, personally i don't like pages asking for permissions instantly but in this case its a necessity. 
What i would like is to commit fully to the library (i'm using it on angular 5), and don't use the Notification API. I'ts more  of a consistency thing. Anyway i would appreciate if you added this feature. Again, thanks for all the work.

**Comment by sagiegurari** [comment]
published version 1.0.28, you can now use:

```js
webNotification.requestPermission(function onRequest(granted) {
    if (granted) {
        console.log('Permission Granted.');
    } else {
        console.log('Permission Not Granted.');
    }
});
```

**Comment by diegotrp** [comment]
It seems to work, thanks for the help.
